test/functional/lua/ui_spec.lua

```diff
@@ -180,5 +180,36 @@ describe('vim.ui', function()
         end, { n.testprg('printargs-test'), 'arg1' })
       )
     end)
+
+    describe('tags in help files', function()
+      local link_ns
+
+      n.command('helptags $VIMRUNTIME/doc')
+
+      before_each(function()
+        n.clear({ args = { '-u', 'NORC' } })
+        n.command('help nvim.txt')
+        link_ns = n.api.nvim_create_namespace('nvim.help.urls')
+      end)
+
+      it('creates extmarks', function()
+        t.neq(0, #n.api.nvim_buf_get_extmarks(0, link_ns, 0, -1, {}))
+      end)
+
+      it('points to https://neovim.io/doc/', function()
```

Comment:
can be a single it()

```suggestion
    it('gx on a help tag opens URL', function()
      n.command('helptags $VIMRUNTIME/doc')
      n.command('help nvim.txt')
      local link_ns = n.api.nvim_create_namespace('nvim.help.urls')

      -- creates extmarks
      t.neq(0, #n.api.nvim_buf_get_extmarks(0, link_ns, 0, -1, {}))
```